index.html

```diff
@@ -14,8 +14,8 @@ <h2>Teachers:</h2>
     <h2>Students:</h2>
     <p>Hello everyone! Please add your name to the list.</p>
     <ul>
-      
+      <li>Zainab sadat</li>
     </ul>
     <hr>
   </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
could you get rid of the whitespace or whatever caused this difference? no worries if not.